kotlin-analysis-api/build.gradle.kts

```diff
@@ -112,7 +112,7 @@ tasks.test {
 
     lateinit var tempTestDir: File
     doFirst {
-        val ideaHomeDir = buildDir.resolve("tmp/ideaHome").takeIf { it.exists() || it.mkdirs() }!!
+        val ideaHomeDir = project.layout.buildDirectory.dir("tmp/ideaHome").get().asFile
```

Comment:
will this be created if it doesn't exist?

Reply:
Ah, no, I thought the `.dir` function will create it automatically, but it's only for task outputs. I've reverted the changes.